catalyst/dl/scripts/run.py

```diff
@@ -64,20 +65,22 @@ def build_args(parser: ArgumentParser):
 
 
 def parse_args():
+    """Parses the command line arguments and returns arguments and config"""
     parser = argparse.ArgumentParser()
     build_args(parser)
     args, unknown_args = parser.parse_known_args()
     return args, unknown_args
 
 
 def main(args, unknown_args):
+    """Run the ``catalyst-dl run`` script"""
     args, config = parse_args_uargs(args, unknown_args)
     set_global_seed(args.seed)
     prepare_cudnn(args.deterministic, args.benchmark)
 
     Experiment, Runner = import_experiment_and_runner(Path(args.expdir))
 
-    runner_params = config.pop("runner_params", {})
+    runner_params = config.pop("runner_params", {}) or {}
```

Comment:
fix for case
```yaml
runner_params: null
```

or

```yaml
runner_params:
```

Reply:
by this way, we should also modify trace script

Reply:
and it is :)